### DISCUSSION TITLE
compiled binary for Linux

### DISCUSSION THREAD

**Discussion by aigenticus** [CarlGao4/Demucs-Gui#89] [Q&A]
I was looking into a compiled binary (inference only) for Linux X86_64 of demucs. Can you please explain how you would go about that? thanks a lot for your help

**Comment by CarlGao4** [comment]
It's quite a complicated process to bundle an AppImage. I'm still finding a method.

**Comment by CarlGao4** [comment]
I've packed binaries for 1.3
